### ISSUE TITLE
The `requirement.txt` file can not be found.

### ISSUE THREAD

**Issue by JiaLonghao1997** [nairouz/CVGAE_PR#2]
In [README.md](https://github.com/nairouz/CVGAE_PR/blob/main/README.md), we found that “All the required libraries are provided in the requirement.txt file.”  

However, the `requirement.txt` file can not be found.

**Comment by JiaLonghao1997** [comment]
I used python 3.9, pytorch  2.2 to run `cora_main.py`, and get `Best accuracy :  0.7226735598227474` after 740 iterations.

![image](https://github.com/nairouz/CVGAE_PR/assets/32414349/d08d67de-7f06-4bd0-a72c-3cee134e5ac9)

Here is the confusion matrix:
![image](https://github.com/nairouz/CVGAE_PR/assets/32414349/52a0ef48-8aa6-43b4-a638-a4f7fbd51642)


Here are my environment from `conda env export > environment_pyg.yml`:
```
name: pyg
channels:
  - pytorch
  - anaconda
  - nvidia/label/cuda-11.8.0
  - nvidia
  - conda-forge
  - bioconda
  - defaults
dependencies:
  - _libgcc_mutex=0.1=conda_forge
  - _openmp_mutex=4.5=2_gnu
  - aiohttp=3.9.3=py39hd1e30aa_1
  - aiosignal=1.3.1=pyhd8ed1ab_0
  - ase=3.22.1=pyhd8ed1ab_1
  - async-timeout=4.0.3=pyhd8ed1ab_0
  - atk-1.0=2.38.0=hd4edc92_1
  - attrs=23.2.0=pyh71513ae_0
  - blas=1.0=mkl
  - blinker=1.7.0=pyhd8ed1ab_0
  - brotli=1.1.0=hd590300_1
  - brotli-bin=1.1.0=hd590300_1
  - brotli-python=1.1.0=py39h3d6467e_1
  - bzip2=1.0.8=hd590300_5
  - ca-certificates=2024.2.2=hbcca054_0
  - cached-property=1.5.2=hd8ed1ab_1
  - cached_property=1.5.2=pyha770c72_1
  - cairo=1.16.0=h35add3b_1015
  - captum=0.6.0=pyhd8ed1ab_0
  - certifi=2024.2.2=pyhd8ed1ab_0
  - charset-normalizer=3.3.2=pyhd8ed1ab_0
  - click=8.1.7=unix_pyh707e725_0
  - cloudpickle=3.0.0=pyhd8ed1ab_0
  - colorama=0.4.6=pyhd8ed1ab_0
  - contourpy=1.2.0=py39h7633fee_0
  - cuda-cudart=11.8.89=0
  - cuda-cupti=11.8.87=0
  - cuda-libraries=11.8.0=0
  - cuda-nvcc=11.8.89=0
  - cuda-nvrtc=11.8.89=0
  - cuda-nvtx=11.8.86=0
  - cuda-runtime=11.8.0=0
  - cudatoolkit=11.8.0=h6a678d5_0
  - cycler=0.12.1=pyhd8ed1ab_0
  - cytoolz=0.12.3=py39hd1e30aa_0
  - dask-core=2024.3.1=pyhd8ed1ab_0
  - expat=2.6.2=h59595ed_0
  - ffmpeg=4.3=hf484d3e_0
  - filelock=3.13.3=pyhd8ed1ab_0
  - flask=3.0.2=pyhd8ed1ab_0
  - font-ttf-dejavu-sans-mono=2.37=hab24e00_0
  - font-ttf-inconsolata=3.000=h77eed37_0
  - font-ttf-source-code-pro=2.038=h77eed37_0
  - font-ttf-ubuntu=0.83=h77eed37_1
  - fontconfig=2.14.2=h14ed4e7_0
  - fonts-conda-ecosystem=1=0
  - fonts-conda-forge=1=0
  - fonttools=4.50.0=py39hd1e30aa_0
  - freetype=2.12.1=h267a509_2
  - fribidi=1.0.10=h36c2ea0_0
  - frozenlist=1.4.1=py39hd1e30aa_0
  - fsspec=2024.3.1=pyhca7485f_0
  - gdk-pixbuf=2.42.10=h05c8ddd_0
  - gettext=0.21.1=h27087fc_0
  - giflib=5.2.1=h0b41bf4_3
  - gmp=6.3.0=h59595ed_1
  - gmpy2=2.1.2=py39h376b7d2_1
  - gnutls=3.6.13=h85f3911_1
  - graphite2=1.3.13=h58526e2_1001
  - graphviz=8.1.0=h28d9a01_0
  - gtk2=2.24.33=h90689f9_2
  - gts=0.7.6=h977cf35_4
  - h5py=3.3.0=nompi_py39h98ba4bc_100
  - harfbuzz=7.3.0=hdb3a94d_0
  - hdf5=1.10.6=nompi_h3c11f04_101
  - icu=72.1=hcb278e6_0
  - idna=3.6=pyhd8ed1ab_0
  - imagecodecs-lite=2019.12.3=py39h44dd56e_7
  - imageio=2.34.0=pyh4b66e23_0
  - importlib-metadata=7.1.0=pyha770c72_0
  - importlib-resources=6.4.0=pyhd8ed1ab_0
  - importlib_metadata=7.1.0=hd8ed1ab_0
  - importlib_resources=6.4.0=pyhd8ed1ab_0
  - intel-openmp=2021.4.0=h06a4308_3561
  - isodate=0.6.1=pyhd8ed1ab_0
  - itsdangerous=2.1.2=pyhd8ed1ab_0
  - jinja2=3.1.3=pyhd8ed1ab_0
  - joblib=1.3.2=pyhd8ed1ab_0
  - jpeg=9e=h166bdaf_2
  - kiwisolver=1.4.5=py39h7633fee_1
  - lame=3.100=h166bdaf_1003
  - lcms2=2.15=hfd0df8a_0
  - ld_impl_linux-64=2.40=h41732ed_0
  - lerc=4.0.0=h27087fc_0
  - libblas=3.9.0=12_linux64_mkl
  - libbrotlicommon=1.1.0=hd590300_1
  - libbrotlidec=1.1.0=hd590300_1
  - libbrotlienc=1.1.0=hd590300_1
  - libcblas=3.9.0=12_linux64_mkl
  - libcublas=11.11.3.6=0
  - libcufft=10.9.0.58=0
  - libcufile=1.9.0.20=0
  - libcurand=10.3.5.119=0
  - libcusolver=11.4.1.48=0
  - libcusparse=11.7.5.86=0
  - libdeflate=1.17=h0b41bf4_0
  - libexpat=2.6.2=h59595ed_0
  - libffi=3.4.2=h7f98852_5
  - libgcc-ng=13.2.0=h807b86a_5
  - libgd=2.3.3=h8faadc7_5
  - libgfortran=3.0.0=1
  - libgfortran-ng=7.5.0=h14aa051_20
  - libgfortran4=7.5.0=h14aa051_20
  - libglib=2.80.0=hf2295e7_1
  - libgomp=13.2.0=h807b86a_5
  - libiconv=1.17=hd590300_2
  - libjpeg-turbo=2.0.0=h9bf148f_0
  - liblapack=3.9.0=12_linux64_mkl
  - liblapacke=3.9.0=12_linux64_mkl
  - libllvm14=14.0.6=hcd5def8_4
  - libnpp=11.8.0.86=0
  - libnsl=2.0.1=hd590300_0
  - libnvjpeg=11.9.0.86=0
  - libpng=1.6.43=h2797004_0
  - librsvg=2.56.3=h98fae49_0
  - libsqlite=3.45.2=h2797004_0
  - libstdcxx-ng=13.2.0=h7e041cc_5
  - libtiff=4.5.0=h6adf6a1_2
  - libtool=2.4.7=h27087fc_0
  - libuuid=2.38.1=h0b41bf4_0
  - libwebp=1.3.2=h11a3e52_0
  - libwebp-base=1.3.2=hd590300_0
  - libxcb=1.13=h7f98852_1004
  - libxcrypt=4.4.36=hd590300_1
  - libxml2=2.11.5=h0d562d8_0
  - libzlib=1.2.13=hd590300_5
  - lightning-utilities=0.11.1=pyhd8ed1ab_0
  - llvm-openmp=15.0.7=h0cdce71_0
  - llvmlite=0.42.0=py39h174d805_1
  - locket=1.0.0=pyhd8ed1ab_0
  - markupsafe=2.1.5=py39hd1e30aa_0
  - matplotlib-base=3.8.3=py39he9076e7_0
  - mkl=2021.4.0=h06a4308_640
  - mkl-service=2.4.0=py39h7e14d7c_0
  - mpc=1.3.1=hfe3b2da_0
  - mpfr=4.2.1=h9458935_0
  - mpmath=1.3.0=pyhd8ed1ab_0
  - multidict=6.0.5=py39hd1e30aa_0
  - munkres=1.1.4=pyh9f0ad1d_0
  - ncurses=6.4.20240210=h59595ed_0
  - nettle=3.6=he412f7d_0
  - networkx=3.2=pyhd8ed1ab_0
  - numba=0.59.0=py39h615d6bd_1
  - numpy=1.22.4=py39hc58783e_0
  - openh264=2.1.1=h780b84a_0
  - openjpeg=2.5.0=hfec8fc6_2
  - openssl=3.2.1=hd590300_1
  - opt_einsum=3.3.0=pyhc1e730c_2
  - packaging=24.0=pyhd8ed1ab_0
  - pandas=2.2.1=py39hddac248_0
  - pango=1.50.14=heaa33ce_1
  - partd=1.4.1=pyhd8ed1ab_0
  - patsy=0.5.6=pyhd8ed1ab_0
  - pcre2=10.43=hcad00b1_0
  - pillow=9.4.0=py39h2320bf1_1
  - pip=24.0=pyhd8ed1ab_0
  - pixman=0.43.2=h59595ed_0
  - psutil=5.9.8=py39hd1e30aa_0
  - pthread-stubs=0.4=h36c2ea0_1001
  - pynndescent=0.5.11=pyhca7485f_0
  - pyparsing=3.1.2=pyhd8ed1ab_0
  - pysocks=1.7.1=pyha2e5f31_6
  - python=3.9.19=h0755675_0_cpython
  - python-dateutil=2.9.0=pyhd8ed1ab_0
  - python-tzdata=2024.1=pyhd8ed1ab_0
  - python_abi=3.9=4_cp39
  - pytorch=2.2.1=py3.9_cuda11.8_cudnn8.7.0_0
  - pytorch-cuda=11.8=h7e8668a_5
  - pytorch-mutex=1.0=cuda
  - pytorch_geometric=2.5.0=pyhd8ed1ab_1
  - pytz=2024.1=pyhd8ed1ab_0
  - pywavelets=1.4.1=py39h44dd56e_1
  - pyyaml=6.0.1=py39hd1e30aa_1
  - rdflib=7.0.0=pyhd8ed1ab_0
  - readline=8.2=h8228510_1
  - requests=2.31.0=pyhd8ed1ab_0
  - scikit-image=0.19.3=py39h4661b88_2
  - scikit-learn=1.4.1.post1=py39ha22ef79_0
  - seaborn=0.13.2=hd8ed1ab_0
  - seaborn-base=0.13.2=pyhd8ed1ab_0
  - setuptools=69.2.0=pyhd8ed1ab_0
  - six=1.16.0=pyh6c4a22f_0
  - statsmodels=0.14.1=py39h44dd56e_0
  - sympy=1.12=pypyh9d50eac_103
  - tabulate=0.9.0=pyhd8ed1ab_1
  - threadpoolctl=3.4.0=pyhc1e730c_0
  - tifffile=2020.6.3=py_0
  - tk=8.6.13=noxft_h4845f30_101
  - toolz=0.12.1=pyhd8ed1ab_0
  - torchaudio=2.2.1=py39_cu118
  - torchmetrics=1.3.2=pyhd8ed1ab_0
  - torchtriton=2.2.0=py39
  - torchvision=0.17.1=py39_cu118
  - tqdm=4.66.2=pyhd8ed1ab_0
  - trimesh=4.2.2=pyhd8ed1ab_0
  - typing-extensions=4.10.0=hd8ed1ab_0
  - typing_extensions=4.10.0=pyha770c72_0
  - tzdata=2024a=h0c530f3_0
  - unicodedata2=15.1.0=py39hd1e30aa_0
  - urllib3=2.2.1=pyhd8ed1ab_0
  - werkzeug=3.0.1=pyhd8ed1ab_0
  - wheel=0.43.0=pyhd8ed1ab_0
  - xorg-kbproto=1.0.7=h7f98852_1002
  - xorg-libice=1.1.1=hd590300_0
  - xorg-libsm=1.2.4=h7391055_0
  - xorg-libx11=1.8.4=h0b41bf4_0
  - xorg-libxau=1.0.11=hd590300_0
  - xorg-libxdmcp=1.1.3=h7f98852_0
  - xorg-libxext=1.3.4=h0b41bf4_2
  - xorg-libxrender=0.9.10=h7f98852_1003
  - xorg-renderproto=0.11.1=h7f98852_1002
  - xorg-xextproto=7.3.0=h0b41bf4_1003
  - xorg-xproto=7.0.31=h7f98852_1007
  - xz=5.2.6=h166bdaf_0
  - yaml=0.2.5=h7f98852_2
  - yarl=1.9.4=py39hd1e30aa_0
  - zipp=3.17.0=pyhd8ed1ab_0
  - zlib=1.2.13=hd590300_5
  - zstd=1.5.5=hfc55251_0
  - pip:
      - scipy==1.10.0
      - torch-cluster==1.6.3+pt22cpu
      - torch-scatter==2.1.2+pt22cpu
      - torch-sparse==0.6.18+pt22cu118
prefix: /home1/jialh/tools/anaconda3/envs/mamba/envs/pyg
```

**Comment by nairouz** [comment]
@JiaLonghao1997 You can reproduce the paper results using the provided pretraining weights. 
Thanks for your interest.
